src/app/retrospective/retrospective-feedback/retrospective-feedback.component.ts

```diff
@@ -436,12 +436,18 @@ export class RetrospectiveFeedbackComponent implements OnInit, OnChanges, OnDest
                 headerName: 'Created By',
                 field: 'CreatedBy',
                 minWidth: 160,
-                valueFormatter: (cellParams) => cellParams.value && (cellParams.value.FirstName + ' ' + cellParams.value.LastName).trim(),
+                cellRenderer: (cellParams) => {
+                    return (cellParams.value.FirstName + ' ' + cellParams.value.LastName).trim();
+                },
                 filter: 'agSetColumnFilter',
+                keyCreator: (cellParams) => {
+                    let key = (cellParams.value.FirstName + ' ' + cellParams.value.LastName).trim();
+                    return key;
```

Comment:
do `return (cellParams.value.FirstName + ' ' + cellParams.value.LastName).trim();` and also create a function to do this so it can be used in other places also.

Reply:
done